ascot-axum/examples/fridge/README.md

```diff
@@ -0,0 +1,45 @@
+# Fridge
+
+## Building
+
+Run the command
+
+```console
+cargo build
+```
+
+## Running the server
+
+The server runs on `localhost` and listens to port `3000`. To make it run:
+
+```console
+cargo run
+```
+
+## REST API examples
+
+Through `curl` or a web browser, it is possible to call the APIs which perform
+some actions on a device.
+
+**Increase temperature**
+
+```console
+curl -X PUT 127.0.0.1:3000/fridge/increase-temperature/1.0
+```
+
+**Decrease temperature**
+
+```console
+curl -X PUT 127.0.0.1:3000/fridge/decrease-temperature/1.0
+```
+
+At the server startup, an initial message signalling its effective execution and
+port number is printed.
+
+Before an action is performed on a device, a message with the REST API
+kind which triggers it is printed.
+
+```
+Starting the Ascot server (port 3000)…
+Performing the REST API (PUT)…
```

Comment:
This is not correct anymore, we should change this part covering the effective behavior